app/controllers/jobs_controller.rb

```diff
@@ -1,5 +1,35 @@
 # frozen_string_literal: true
 
 class JobsController < ApplicationController
-  def index; end
+  skip_before_action :verify_authenticity_token, only: :update_multiple
+
+  def index
+    return order_jobs if params[:job_ids]
+    @date = params[:job_date] || l(Date.current + 2.days)
```

Comment:
Die Vorlaufzeit von 2 Tagen bitte aus einem Wert in der `settings.yml` laden, das sollte besser von Anfang an konfigurierbar sein!